src/Bundle/spec/Doctrine/PHPCRODM/DataSourceSpec.php

```diff
@@ -103,7 +103,11 @@ function it_should_get_the_data(
         $query->setFirstResult(Argument::any())->willReturn($query);
         $query->execute()->willReturn([]);
 
-        $this->getData(new Parameters(['page' => 1]))->shouldHaveType(Pagerfanta::class);
+        $data = $this->getData(new Parameters(['page' => '1']));
+
+        $data->shouldHaveType(Pagerfanta::class);
+        $data->getCurrentPage()->shouldReturn(1);
+        $data->getNormalizeOutOfRangePages()->shouldReturn(false);
```

Comment:
@Zales0123 It has "false" value, is that normal?

Reply:
This is current state of this data source, so I would leave it as is. Probably no one is using it or the current behaviour is ok, as there weren't any issues about it. We could also skip this check in this case

Reply:
I think we can leave this test to be aware of the current implementation.